### DISCUSSION TITLE
Parent Accounts

### DISCUSSION THREAD

**Discussion by gmitchell1982** [blueseerERP/blueseer#75] [Q&A] [answered]
I have been trying to set up a chart of accounts, but have been unable to figure out how to utilize parent (or placeholder) accounts in the accounting system (example: so I can keep sales tax collected in the same physical account as the rest of my money but in a different accounting account and still be able to reconcile balances). I did not see anything in the BlueSeer Manual. Just wondering if there is a way I am missing on how to set up such a hierarchical account tree?

Thank you for you time.

**Comment by vaughnte** [comment, marked as answer]
Unfortuanately, there is no hierarchical structure mechanism built within the GL Chart of Account table.   The ac_mstr table is a 'flat' table where all account records are discrete peer-based objects.   Other than using group-related account numbers, as is done for the default AP Sales Tax accounts ex: 34xxxxxx series, there's no other way to build parent/child relationships with the account numbers.

**Reply by gmitchell1982** [reply]
Thank you for the response.
